src/gui/wizard/webview.cpp

```diff
@@ -145,7 +147,12 @@ QWebEnginePage * WebEnginePage::createWindow(QWebEnginePage::WebWindowType type)
     ExternalWebEnginePage *view = new ExternalWebEnginePage(this->profile());
     return view;
 }
-
+    
+bool WebEnginePage::certificateError(const QWebEngineCertificateError &certificateError) {
+    //handle webview certificate errors here 
+return true;
```

Comment:
Please, fix the indentation.

Reply:
This means we just accept all self signed certificates right?

Reply:
certificates are already accepted or rejected from the setup page.